multiclusterhub/charts/search-prod/templates/api-role-binding.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2020 Red Hat, Inc.
+# Copyright Contributors to the Open Cluster Management project
+
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ .Values.org }}:{{ .Release.Name }}:search-api
+subjects:
+- kind: ServiceAccount
+  name: search-api
+  namespace: {{ .Release.Namespace }}
+roleRef:
+  kind: ClusterRole
+  name: {{ .Values.org }}:{{ .Release.Name }}:search-api
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
Missing newline at end of last line.

https://stackoverflow.com/a/729795/1437822